### ISSUE TITLE
Terra-tabs v7.0.0 and above doesn't display on initial render

### ISSUE THREAD

**Issue by ms010922** [cerner/terra-framework#1962]
# Bug Report

## Description
After upgrading to version 7.0.0 of terra-tabs it was discovered that the tabs for the Ongoing and Resolved tables were not displayed in the component.  After further investigation it was determined that with the introduction of common-tabs with version 7.0.0 and above the CSS class data-tab-is-calculating now has a "visibility:hidden" attribute that is removed once another control in the DOM is selected.  All the details with video and screenshots are provide in the following Jira.

[https://jira2.cerner.com/browse/PROBDIAG-23237](url)

## Steps to Reproduce
1. Login to M18DV PowerChart with user PROBLEMSA/cerner
2. Open chart of test patient PrivCondOR, Con Elam
3. Navigate to the Provider View tab in the PowerChart Table of Contents
4. Click on the Inpatient Workflow tab in the viewpoint
5. The Cloud Problem List component will render without the Ongoing and Resolved tabs
6. In the View Navigation pane click on the Problem List component or click on another view in the Viewpoint and come bac to the Inpatient Workflow view
7. The Ongoing and Resolved tabs are now visible.

## Additional Context / Screenshots
All details are in the provided JIRA above.

## Expected Behavior
The Ongoing and Resolved tabs should be visible on initial render of the component.

## Possible Solution
The visibility:hidden attribute should not be set by default on this class.

## Environment
* Component Name and Version: Problem List v5.2.0
* Browser Name and Version:  Internet Explorer 11 it is important to note that we do not see this behavior in Edge
* Node/npm Version: [e.g. Node 8/npm 5] Node 14/npm 6
* Webpack Version: "webpack": "^4.35.2",
* Operating System and version (desktop or mobile): Windows 10 desktop

## @ Mentions
N/A
